rmw_connext_shared_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(rmw_connext_shared_cpp)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(connext_cmake_module REQUIRED)
+find_package(Connext MODULE)
+if(NOT Connext_FOUND)
+  ament_package()
+  message(WARNING "Could not find RTI Connext - skipping '${PROJECT_NAME}'")
+  return()
+endif()
+find_package(rmw REQUIRED)
+
+include_directories(include)
+
+ament_export_definitions(${Connext_DEFINITIONS})
+ament_export_dependencies(rmw)
+ament_export_include_directories(${Connext_INCLUDE_DIRS})
```

Comment:
from what I can tell by looking at how `ament_export_include_directories`, this command is for exporting external directories that are included in this package. I essentially copied this pattern from the CMakeLists of rmw_connext_cpp on master, which calls `ament_export_include_directories(${Connext_INCLUDE_DIRS})` but not on its own `include` directory: https://github.com/ros2/rmw_connext/blob/master/rmw_connext_cpp/CMakeLists.txt